### DISCUSSION TITLE
Release package checksum not match

### DISCUSSION THREAD

**Discussion by toanalien** [ConsenSys/quorum#1454] [Q&A] [answered]
In release v22.4.4 checksum of all package not match.

| Package                          | Expect                                                           | Real                                                             |
|----------------------------------|------------------------------------------------------------------|------------------------------------------------------------------|
| geth_v22.4.4_linux_amd64.tar.gz  | 306b224667a2ab43876a0182ddc109c98866694b02b3684a6ddfcae3674f015c | cf13377f33b03754f0702e2c184e51e4d0b2450842a4462b3f5151354b4392ef |
| geth_v22.4.4_darwin_amd64.tar.gz | 9ae7c3a15ae8a3f9c9133c16ee1d0be000d2b5544191604a03f09e70293d939e | 829eaef5e7a62224c883ccd98c6282dc2bb43188230371be2c33976715700a10 |

**Comment by antonydenyer** [comment, marked as answer]
~~Thanks @toanalien I suspect this is because we had a problem with the release - you can see we ran it twice. https://github.com/ConsenSys/quorum/actions/workflows/release.yml?query=branch%3Av22.4.4 - let me investigate and I'll correct accordingly.~~

I've rebuilt binaries and docker image, and can confirm the following:



Filename | SHA256 Hash
-- | --
geth_v22.4.4_linux_amd64.tar.gz | 89d1119e4a7e0e841d1d3476cf2c5811b558de0284f51f52c008981238085bef
geth_v22.4.4_darwin_amd64.tar.gz | 9b0529a66840a505a725a541ae42039ab46c3a652d39c117cfda0c58d5620f9f

thanks again @toanalien great spot.
